ipad/Utilities/Core/Extensions/UIViewControllerExtensions.swift

```diff
@@ -145,3 +145,21 @@ extension UIViewController {
         return alert
     }
 }
+
```

Comment:
This helps show alerts regardless of navigation stack (helped to display our alert box in each window)